example/pythonlib/module/5-resources/foo/test/src/test.py

```diff
@@ -0,0 +1,36 @@
+import os
+from pathlib import Path
+import importlib.resources
+import unittest
+from foo import Foo  # type: ignore
+
+
+class TestScript(unittest.TestCase):
+    def test_all(self) -> None:
+        appClasspathResourceText = Foo().PythonPathResourceText("resources", "file.txt")
```

Comment:
minor: maybe avoid the term "classpath", since it might be unfamiliar to python devs. I think "path" or "pythonpath" is fine

Reply:
Sorry it's just like typo coz i focused on code logics only and more tuning will be done on the go 
I'll update

Reply:
done!